cleanup.sh

```diff
@@ -1,25 +1,42 @@
+#!/bin/bash
 #remove directory
 
 cd ~
 
-rm -rf antora 
+rm -rf antora/ 
 
 sudo rm -rf /var/www/html/*
+ 
+#nodejs, npm, nvm, antora cli, antora site generator, http server, apache2, popler-utils, pandoc, python-pdfminer 
 
-#@todo -uninstallation of dependencies 
-#nodejs, npm, nvm, antora cli, antora site generator, http server, apache2 
+sudo npm uninstall -g @antora/site-generator-default
 
+sudo npm uninstall -g http-server
 
-sudo apt remove nodejs
+npm cache clean --force
 
-sudo apt purge nodejs
+sudo apt-get remove npm
+sudo apt autoremove
```

Comment:
i see you have autoremove on here twice, is that necessary and what does this actually do?

Reply:
Autoremove will remove any unused dependencies that a package requires, I had added it after each applications so I could ensure each package's dependencies were removed with it. This could likely be placed once at the end of the script and achieve the same goal.

Reply:
@kgm78 for purposes of keeping code DRY, let's try executing `apt autoremove` at the end of the cleanup script.

Reply:
@kgm78 
Also is it possible to combine all the apt-get calls into one command?

Reply:
Yes, I just pushed a update which puts all apt-get removes on one line and references auto remove only once. Does this fix the issues?